README.md

```diff
@@ -27,9 +31,22 @@ Metrics will now be accessible at http://localhost:8080/metrics
 
 To bind the java agent to a specific IP change the port number to `host:port`.
 
-See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
+__Note:__ The configuration file **must not** specify _hostPort_ or _jmxUrl_ when running in agent mode.
 
-Please note that due to the nature of JMX the `/metrics` endpoint might exceed Prometheus default scrape timeout of 10 seconds.
+### Standalone mode
+
+To run in standalone mode, download one of the JARs and run:
+```
+java -javaagent:jmx_prometheus_javaagent/target/jmx_prometheus_javaagent-0.16.2-SNAPSHOT.jar=8080:example_configs/httpserver_sample_config.yml -jar integration_tests/jmx_example_application/target/jmx_example_application.jar
```

Comment:
The `-javaagent` flag is for running it in agent mode, i.e. attaching the agent directly to the JVM at startup. In standalone mode you would run the application without the agent, but with a JMX remote port exposed, and then connect the `jmx_exporter` to that port.

Reply:
Good catch. Mistake on my part. Will try to fix it this weekend.